src/main/java/usecases/CourseEvaluatorUseCase/EvaluatorDbRequestModel.java

```diff
@@ -1,25 +1,40 @@
 package usecases.CourseEvaluatorUseCase;
 
 public class EvaluatorDbRequestModel {
-    private final String studentID;
-    private final String courseID;
-    private final float grade;
-
-    public EvaluatorDbRequestModel(String studentID, String courseID, float grade) {
-        this.studentID = studentID;
-        this.courseID = courseID;
-        this.grade = grade;
-    }
+    private String studentID;
+    private String password;
+    private String courseID;
+    private float grade;
 
     public String getStudentID() {
         return studentID;
     }
 
+    public String getPassword() {
+        return password;
+    }
+
     public String getCourseID() {
         return courseID;
     }
 
     public float getGrade() {
         return grade;
     }
+
+    public void setStudentID(String studentID) {
+        this.studentID = studentID;
+    }
+
+    public void setPassword(String password) {
+        this.password = password;
+    }
+
+    public void setCourseID(String courseID) {
+        this.courseID = courseID;
+    }
+
+    public void setGrade(float grade) {
+        this.grade = grade;
+    }
 }
```

Comment:
I see that you updated the dbrequestmodel to contain all that is necessary for potentially creating a new row. Well done!